MichaelAng/dnd-char-gen
# Feature/39/add basic header

# Issue

PR #39 
# Description

Created a basic header skeleton.  
# Changes
- Created a basic header skeleton
# Steps to test
1. run `docker-compose build && docker-compose start`
2. [x] Verify it says hello world
# Help

[wiki](https://github.com/MichaelAng/dnd-char-gen/wiki)
